examples/translation_data_generation_example/wikipedia/qna.yaml

```
@@ -0,0 +1,74 @@
+    The move helps accelerate plans for the company to widen the launch of its Binance Card, which will enable users to buy goods and services using cryptocurrency such as Bitcoin from outlets not normally used to trading in digital assets.
+    Binance has run into problems implementing its own card idea, so Swipe’s knowledge of the debit card industry and how it ticks could help push the Binance Card onwards past the concept stage.
+    Ultimately, Binance hopes the purchase of Swipe will take cryptocurrencies to a more mainstream market. The collaboration could end up bridging the gap between fiat and digital assets by making payments and purchases in cryptocurrency accessible via traditional financial systems.
+    Swipe is undoubtedly a good match as its combination of digital wallet and Visa debit card features lets users buy, sell and convert cryptocurrencies. Users of Swipe can already pay with cryptocurrency in selected stores around the globe.
+    “To achieve our mission of making crypto more accessible to the masses, off-ramps are a key component as well. By giving users the ability to convert and spend crypto directly, and have merchants still seamlessly accept fiat, this will make the crypto experience much better for everyone,” said Binance CEO Changpeng Zhao.
+    Swipe has a growing profile in 31 countries across Europe and supports fiat transactions in Euros, British Pounds, US Dollars, Korean Won and Philippine Pesos. It has additional plans in place to launch in both Asia and North America.
+    Zhao, also known as CZ, recently highlighted the potential of payments using crypto in his Twitter feed, but conceded that with only 0.1% of people having crypto there was no real incentive for merchants to accept it. The CEO obviously hopes that buying Swipe will enable Binance to address the issue and incentivise merchants to become more open to accepting cryptocurrency.
+    Sign up to the TechRadar Pro newsletter to get all the top news, opinion, features and guidance your business needs to succeed!
+    Rob Clymo has been a tech journalist for more years than he can actually remember, having started out in the wacky world of print magazines before discovering the power of the internet. Since he's been all-digital he has run the Innovation channel during a few years at Microsoft as well as turning out regular news, reviews, features and other content for the likes of TechRadar, TechRadar Pro, Tom's Guide, Fit&Well, Gizmodo, Shortlist, Automotive Interiors World, Automotive Testing Technology International, Future of Transportation and Electric & Hybrid Vehicle Technology International. In the rare moments he's not working he's usually out and about on one of numerous e-bikes in his collection.
+
+  questions_and_answers:
+    - question: What is the significance of Binance acquiring Swipe, and how does it align with Binance’s goals?
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 2 but found 4

(indentation)